habitat-packages/chef-client/plan.sh

```diff
@@ -2,7 +2,7 @@ pkg_name=chef-client
 pkg_origin=chef
 pkg_maintainer="The Chef Maintainers <humans@chef.io>"
 pkg_description="The Chef Client"
-pkg_version=$(cat ../VERSION)
+pkg_version=$(cat ../../VERSION)
```

Comment:
Not a blocker, but I'd prefer we use the update_version function (https://github.com/chef/ci-studio-common/blob/master/habitat/plan.sh#L10-L17), or use Expeditor to update the version inline using sed (https://github.com/chef/expeditor/blob/master/.expeditor/update_version.sh#L7).

Reply:
Whoa. Using pkg_version() like that is really dope and intuitive.

Reply:
Fixed! I went with the pkg_version() route.